src/crewai/cli/create_crew.py

```
@@ -1,80 +1,123 @@
 import os
 from pathlib import Path
-
 import click
-
-
-def create_crew(name):
-    """Create a new crew."""
+import shutil
```

Ruff

`shutil` imported but unused

Remove unused import: `shutil`

(F401)